### PR TITLE
fix(prayer): trigger pray and navigate to prayer submit after last pr…

### PR DESCRIPTION
**Pull request by dsilva609**
## ✏️ Description
- trigger pray and navigate to prayer submit after last prayer

## 🔬 To Test
1. Open up the Prayer Experience
2. Pray/scroll through ALL prayers in the list until you get to the very last one
3. Press the "pray" button and you should continue to the prayer submit screen
4. 

## 📝 Checklist

- [x] I have performed a self-review of my code
- [x] I have tested my code works in the project (nothing breaks)
- [x] I have commented my code in hard-to-understand areas
- [x] New and existing tests and linting pass locally with my changes
- [x] I have made corresponding changes to the documentation (new envs?)
- [x] For PR's with design updates/changes, I have consulted with the product designer and made all necessary adjustments in Figma

### CODE REVIEW

File: templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js [hunk 1]
```diff
@@ -146,16 +146,21 @@ const PrayerExperienceSwiper = ({}) => {
   }, []);
 
   function onPress() {
-    if (selectedIndex >= totalCount - 1 || !flatListRef || !prayerRequestId) {
+    if (selectedIndex > totalCount - 1 || !flatListRef || !prayerRequestId) {
```
Comment (calebpanza) [inline templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js:149]: 🤦🏻‍♂️ that's my bad... so glad it was something simple!

File: templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js [hunk 2]
```diff
@@ -146,16 +146,21 @@ const PrayerExperienceSwiper = ({}) => {
...
 
     pray({
       variables: {
         prayerRequestId,
       },
-      onCompleted: () =>
-        flatListRef.current.scrollToIndex({ index: selectedIndex + 1 }),
+      onCompleted: () => {
+        if (selectedIndex + 1 === totalCount) {
+          return navigate('UserInputScreen');
```
Comment (calebpanza) [inline templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js:159]: I'm not sure that this is the final action that we want to happen, but I like that it's in here and we'll make sure to get specific feedback while testing on device if this interaction feels "correct" to users.

**Review by calebpanza — APPROVED** [review]
Checked on both iOS and Android and everything is working as expected!

File: templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js (hunk 2 above)
Comment (dsilva609) [reply templates/mobile/src/components/PrayerExperienceSwiper/PrayerExperienceSwiper.js:159]: Yeah I wasn't sure what to do after everything was run through. I could be another screen that just says "All prayers prayed for" or something but this was the simplest solution for now.

**Pull request merged**